.eslintrc.js

```diff
@@ -1,65 +1,29 @@
 module.exports = {
-    env: {
-        node: true,
-        es6: true,
+  env: {
+    node: true,
+    es6: true,
+  },
+  parser: "babel-eslint",
+  parserOptions: {
+    sourceType: "module",
+  },
+  settings: {
+    react: {
+      createClass: "createReactClass",
+      pragma: "React",
+      version: "detect",
+      flowVersion: "0.53",
     },
-    extends: ["eslint:recommended", "plugin:react/recommended"],
-    parser: "babel-eslint",
-    parserOptions: {
-        sourceType: "module",
-    },
-    "settings": {
-        "react": {
-            "createClass": "createReactClass",
-            "pragma": "React",
-            "version": "detect",
-            "flowVersion": "0.53"
-        },
-        "linkComponents": [
-            {"name": "Link", "linkAttribute": "to"},
-            {"name": "OutboundLink", "linkAttribute": "to"}
-        ]
-    },
-    rules: {
```

Comment:
We will let prettier take care of formatting rules - thus remove all of them from this file